packages/connector-facebook/README.md

```diff
@@ -1,2 +1,52 @@
-### FB Social Connector README
-placeholder
+# Facebook
+
+The Facebook connector provides a concise way for your application to use Facebook's OAuth 2.0 authentication system.
+
+## Register a Facebook developer account
+
+[Register as a Facebook Developer](https://developers.facebook.com/docs/development/register/) if you don't already have one
+
+## Set up a Facebook app
+
+1. Visit the [Apps](https://developers.facebook.com/apps) page.
+2. Click your existing app or [create a new one](https://developers.facebook.com/docs/development/create-an-app) if needed.
+3. On the app dashboard page, scroll to the **Add a product** section and click the **Set up** button on the **Facebook Login** card.
+4. Skip the Facebook Login Quickstart page, and click the left sidebar -> **Products** -> **Facebook Login** -> **Settings**.
+5. In the Facebook Login Settings page, fill `${your_logto_origin}/callback/facebook-universal` in the **Valid OAuth Redirect URIs** field. E.g.:
+    - `https://logto.dev/callback/facebook-universal` for production
+    - `https://localhost:3001/callback/facebook-universal` for testing in the local environment
+6. Click the **Save changes** button at the bottom right corner.
+
+## Set up the Logto connector settings
+
+1. In the Facebook app dashboard page, click the left sidebar -> **Settings** -> **Basic**.
+2. You will see the **App ID** and **App secret** on the right column.
+3. Click the **Show** button following the App secret input box to copy its content.
+4. Fill out the Logto connector settings:
+    - Fill out the `clientId` field with the string from **App ID**.
+    - Fill out the `clientSecret` field with the string from **App secret**.
+
+## Debug sign-in with the test users
```

Comment:
I was wondering if I can dive into testing if I skip this part?
If the answer is no, I don't think this part should be placed in this position.

Reply:
Should be placed where?

Reply:
Another teammate maybe tries to test sign-in with their own Facebook account, and then they will fail.

So I think it's necessary to tell how to test sign-in with the Facebook app and why.

Added explanation:

```ts
You can also [take the app live](#take-the-facebook-app-live) directly so that any Facebook user can test sign-in with the app.
```

PTAL